src/UKHO.ERPFacade.Common/Helpers/IAzureTableStorageHelper.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Newtonsoft.Json.Linq;
+using UKHO.ERPFacade.Common.Models.TableEntities;
+
+namespace UKHO.ERPFacade.Common.Helpers
+{
+    public interface IAzureTableStorageHelper
+    {
+        Task UpsertEntity(JObject eesEvent, string traceId, string correlationId);
+        Task<EESEventTable?> GetEntity(string traceId);
```

Comment:
Do we need Task<EESEventTable?> nullable ?

Reply:
Removed as it is not needed.